gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -527,6 +527,11 @@
 
   public static final String SQL_SERVER_CONNECTION_PARAMETERS = "source.querybased.sqlserver.connectionParameters";
 
+  /**
+   * Configuration properties used by the CopySource.
+   */
+  public static final String COPY_SOURCE_FILESET_WU_GENERATOR = "copy.source.fileset.wu.generator";
```

Comment:
We usually have `class` at the end of key name, so maybe `copySourceFilesetWorkunitGenerator.class` or `copy.source.fileset.workunit.generator.class`?